tests/phpunit/CRM/Contact/Import/Parser/ContactTest.php

```diff
@@ -1863,9 +1868,11 @@ public function testImportParserDoesNotMatchPrimaryToRelated(): void {
 
     $parser = new CRM_Contact_Import_Parser_Contact($fields);
     $parser->setUserJobID($userJobID);
-    $parser->init();
+    $dataSource = new CRM_Import_DataSource_CSV($userJobID);
 
-    $this->assertEquals(CRM_Import_Parser::ERROR, $parser->import(CRM_Import_Parser::DUPLICATE_UPDATE, $values), 'Return code from parser import was not as expected');
+    $parser->init();
+    $parser->import(CRM_Import_Parser::DUPLICATE_UPDATE, $values);
+    $this->assertEquals(1, $dataSource->getRowCount([CRM_Import_Parser::ERROR]));
```

Comment:
The other tests already use this method - ie check the actual import table not the return code (which there no longer is)